package.json

```diff
@@ -88,7 +88,7 @@
     "react-native": "0.63.4",
     "react-native-builder-bob": "^0.18.0",
     "react-native-macos": "^0.63.4",
-    "react-native-test-app": "^0.9.0",
+    "react-native-test-app": "^0.9.5",
```

Comment:
@tido64 I've noticed that `react-native-test-app` is overriding `kotlinVersion` from root project, which leads to build fails if I override it myself. Looked up the release history of test-app, I updated to 0.9.5, where Kotlin was bumped to desired version.

Are there any extra steps needed to this version bump?

Reply:
No, it should just work. Are you seeing issues?

Reply:
Nope, just making sure 😄